src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,45 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int DEFAULT_LENGTH_FOR_LOGIN_AND_PASSWORD = 6;
```

Comment:
let's break it into two separate constants, if the requirement changes for login (for example) - we should be able to make changes to our codebase easily

Reply:
i agee and i thought about this, will do!

Reply:
i agree++